File: tests/project_multilang/swift/expected/Caller.swift
func run(_ c: Any) {
  (c as! ICallback).onReady()
  if (c as! ICallback).isEnabled() {
    print("on")
  }
}

func multiline(flag: Bool) {
  use(false)
}

func use(_ b: Bool) { print(b) }

File: tests/project_multilang/kotlin/expected/Caller.kt
package demo
fun run(c: Any) {
  (c as ICallback).onReady()
  if ((c as ICallback).isEnabled()) {
    println("on")
  }
}

fun multiline(flag: Boolean) {
  use(false)
}

fun use(b: Boolean) { println(b) }
